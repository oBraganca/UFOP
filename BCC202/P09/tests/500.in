500
Yago 83271 1.6
Miguel 463864 0.9
Emanuel 254280 2.1
Lucca 673398 2.4
Emilly 575184 9.8
Theo 215 3.0
Liz 110127 7.7
AnaSophia 757983 4.2
AnaJulia 258679 6.3
MariaVitoria 485389 1.2
Vitoria 468010 1.8
Luan 887924 7.1
Lorena 897218 3.8
Bernardo 729396 3.7
MariaHelena 62255 4.5
ArthurHenrique 542641 2.1
Marina 447724 8.3
Gael 156161 0.9
Isabella 450496 6.8
Helena 191184 6.4
Antonella 270313 9.7
Sophia 437249 3.7
DaviLucca 605485 5.6
JoaoMiguel 268427 3.0
MariaLaura 879273 2.5
Raul 192002 9.0
DaviLucas 201464 9.2
LuizMiguel 375774 1.0
Bryan 14110 3.8
Louise 563521 7.5
Carolina 320126 7.9
AnaVitoria 718288 4.9
Augusto 638511 5.8
ArthurMiguel 76880 4.1
Felipe 268577 4.1
Alicia 762996 8.1
MariaClara 950345 8.0
Elisa 729894 9.0
Kauê 85760 2.1
Arthur 365522 1.2
Thomas 541886 8.7
Eloa 36760 5.6
Thiago 953881 5.2
Agatha 775468 3.6
Ana 1734 6.7
Isadora 702044 8.1
Caio 507040 5.2
Larissa 529489 5.6
MariaHeloisa 868477 4.0
Gabriel 631960 1.4
Mariana 928332 9.1
Stella 825 1.3
LuizFelipe 21333 3.3
Eduardo 969596 4.0
Samuel 373314 3.6
Caua 846842 4.2
Allana 509833 2.5
Danilo 504032 9.5
MariaIsis 822458 8.7
Breno 911259 3.6
Henrique 664983 5.6
Vinicius 208375 4.0
Ryan 182371 3.2
Noah 199962 6.8
LuizHenrique 904421 4.9
Isabel 306940 0.6
Beatriz 105783 1.3
Lavinia 114098 5.8
Martin 944273 4.9
AnaBeatriz 571350 1.0
Bruna 367688 5.8
Miguel 50477 7.7
Lorenzo 508404 1.0
Heitor 76947 3.4
Rodrigo 965140 1.2
Laura 564090 9.0
JoaoPedro 27084 8.9
Enrico 576215 5.5
Heloisa 318507 2.0
Julia 703209 8.0
Murilo 304314 4.1
Fernanda 501011 5.7
Valentina 817300 4.7
Gabrielly 120621 2.7
ArthurGabriel 285872 6.8
Lara 12183 7.1
Bento 683393 9.9
Joaquim 532999 4.2
Benjamin 88894 2.6
Leticia 581872 4.6
Guilherme 697944 7.7
Tomas 182589 4.0
Anthony 569381 1.7
Sarah 97615 3.1
Rafael 449601 6.3
MariaFernanda 688453 9.1
Gabriela 182231 1.4
Kaique 300183 4.5
JoaoVictor 810344 5.2
Clara 925424 5.3
Andre 281154 2.4
Gabriel 292331 6.9
Rebeca 836059 8.0
Manuela 595399 7.1
Maria 474286 5.6
Ruan 606890 7.1
MariaAlice 590594 6.0
Benicio 822726 5.5
Luana 471718 1.3
DaviMiguel 76260 5.6
Levi 749091 5.1
Lucas 320485 8.3
AnaLivia 410806 7.7
Sophie 795858 4.1
Antonio 389671 1.0
Oliver 849521 2.2
Clarice 641685 8.9
Erick 949111 8.6
Joana 947751 8.8
AnaCecilia 744093 6.6
Gustavo 595974 8.0
Milena 260906 2.0
LuizOtavio 197081 1.9
Davi 760183 8.3
Luiza 104507 6.8
Nathan 531058 5.7
Analu 867155 1.6
JoaoGabriel 855921 8.0
DaviLuiz 342650 4.3
Fernando 18652 3.1
CarlosEduardo 243465 6.3
Henry 558152 7.4
Lais 281968 3.8
Caleb 198040 6.2
EnzoMiguel 280455 6.3
Matheus 318725 8.8
MariaJulia 479915 2.1
Ayla 7034 3.3
Amanda 871304 2.7
VictorHugo 960395 1.5
Catarina 314370 0.9
Heloise 141848 8.2
Joao 568961 7.4
AnaLaura 665930 3.1
Jose 913945 5.0
Enzo 978062 1.8
Yasmin 347740 4.9
Pietra 591628 9.5
PedroHenrique 323568 9.9
Leonardo 158253 2.0
LuizGustavo 807041 5.2
EnzoGabriel 686587 3.6
Emanuelly 887006 5.5
Aurora 940893 5.8
Vicente 285634 5.0
MariaEduarda 97285 7.8
MariaSophia 481058 0.9
Bruno 416267 9.0
LucasGabriel 198874 4.1
JoaoLucas 217875 1.2
Anthony 22709 1.6
Ian 474641 9.7
PedroLucas 917197 5.1
Yuri 390776 5.6
Alice 631097 9.3
Nicolas 191832 4.0
MariaLuiza 892017 8.6
MariaCecilia 215556 3.4
Otavio 800109 2.3
Bianca 584717 0.6
AnaLuiza 746474 6.6
PedroMiguel 133326 4.1
Pedro 971657 4.3
MariaValentina 229209 9.8
Giovanna 249151 4.8
Cecilia 981018 7.5
Esther 916511 2.8
Isabelly 31069 6.9
Livia 656287 7.8
Nicole 398592 5.2
AnaLiz 765124 9.5
Malu 641153 8.8
JoaoGuilherme 684578 4.0
Isaac 215812 5.1
Francisco 263168 4.4
Daniel 789850 5.2
Maitê 19930 6.3
Olivia 402614 3.6
Eduarda 174322 6.5
Victor 579030 3.3
Pietro 828771 7.7
Maya 985540 8.3
MariaFlor 139592 1.5
Antonia 408719 2.9
Melissa 984326 7.0
Mirella 566018 1.5
Henrique 400175 7.6
Rafaela 857337 3.8
AnaClara 499549 2.0
Luna 718735 8.1
Mariah 172403 1.2
Isis 506111 9.1
Yago 121352 6.7
Miguel 93098 4.8
Emanuel 360692 7.6
Lucca 366467 9.2
Emilly 466521 6.9
Theo 142115 6.3
Liz 41739 6.2
AnaSophia 274017 2.6
AnaJulia 529241 8.2
MariaVitoria 534268 2.5
Vitoria 396525 3.1
Luan 949278 6.6
Lorena 881917 6.7
Bernardo 974729 10.0
MariaHelena 809404 8.9
ArthurHenrique 986304 1.2
Marina 783655 0.9
Gael 252232 1.9
Isabella 859251 4.2
Helena 350832 4.2
Antonella 259195 3.3
Sophia 557268 5.5
DaviLucca 605502 0.8
JoaoMiguel 483528 4.3
MariaLaura 380292 4.7
Raul 853197 2.9
DaviLucas 457249 8.6
LuizMiguel 407826 3.8
Bryan 68875 2.3
Louise 870632 9.6
Carolina 622297 2.9
AnaVitoria 315378 2.6
Augusto 396061 9.3
ArthurMiguel 332721 3.1
Felipe 516131 8.5
Alicia 796322 1.3
MariaClara 275020 1.1
Elisa 11187 9.9
Kauê 723631 5.5
Arthur 55242 9.1
Thomas 763560 5.3
Eloa 412970 8.2
Thiago 127497 4.0
Agatha 609677 10.0
Ana 765971 8.8
Isadora 548376 6.1
Caio 247076 1.0
Larissa 671603 7.2
MariaHeloisa 327673 3.4
Gabriel 767040 6.6
Mariana 344745 7.9
Stella 635060 7.5
LuizFelipe 85013 4.2
Eduardo 789299 9.5
Samuel 881887 9.1
Caua 743841 5.7
Allana 345012 5.1
Danilo 894641 9.5
MariaIsis 784649 6.1
Breno 521330 7.7
Henrique 199728 9.4
Vinicius 686440 9.7
Ryan 296821 3.4
Noah 691971 7.7
LuizHenrique 42263 2.8
Isabel 623833 1.8
Beatriz 49072 1.8
Lavinia 838900 8.3
Martin 306197 8.0
AnaBeatriz 209835 1.7
Bruna 259800 6.6
Miguel 75109 7.8
Lorenzo 366023 8.5
Heitor 540726 1.0
Rodrigo 960450 7.5
Laura 767909 3.1
JoaoPedro 422574 5.2
Enrico 299318 1.3
Heloisa 758760 5.7
Julia 38547 3.6
Murilo 12318 3.2
Fernanda 578638 2.3
Valentina 773133 9.0
Gabrielly 444280 4.9
ArthurGabriel 250328 7.3
Lara 736579 5.2
Bento 841209 5.7
Joaquim 760615 3.0
Benjamin 966632 4.6
Leticia 330354 8.3
Guilherme 78275 7.9
Tomas 214777 4.0
Anthony 853302 1.5
Sarah 620648 3.0
Rafael 535828 8.6
MariaFernanda 582695 4.6
Gabriela 52415 5.6
Kaique 47101 0.7
JoaoVictor 88463 8.8
Clara 954235 2.3
Andre 737240 9.2
Gabriel 191698 6.6
Rebeca 255848 6.1
Manuela 401810 7.1
Maria 200853 1.5
Ruan 295172 8.7
MariaAlice 430138 6.6
Benicio 929061 8.9
Luana 926077 1.4
DaviMiguel 380589 5.0
Levi 926862 8.9
Lucas 193236 2.3
AnaLivia 36809 9.0
Sophie 57449 2.0
Antonio 510353 7.3
Oliver 471808 5.8
Clarice 231717 7.4
Erick 196948 5.4
Joana 471655 7.1
AnaCecilia 100446 7.0
Gustavo 842329 7.6
Milena 818216 3.9
LuizOtavio 411197 8.9
Davi 143083 8.1
Luiza 861525 3.4
Nathan 670471 0.8
Analu 666012 3.0
JoaoGabriel 601303 8.9
DaviLuiz 617745 4.2
Fernando 622886 8.6
CarlosEduardo 818253 5.7
Henry 570628 9.0
Lais 255580 0.8
Caleb 337450 4.4
EnzoMiguel 751973 3.3
Matheus 840003 2.3
MariaJulia 779846 5.6
Ayla 932012 2.4
Amanda 279304 3.0
VictorHugo 377592 8.2
Catarina 139381 4.5
Heloise 263375 1.0
Joao 753046 1.6
AnaLaura 362626 8.0
Jose 158805 6.6
Enzo 848100 9.5
Yasmin 243147 7.7
Pietra 532292 2.6
PedroHenrique 682653 5.0
Leonardo 290864 3.0
LuizGustavo 366151 9.5
EnzoGabriel 419192 5.3
Emanuelly 132580 6.8
Aurora 6519 3.5
Vicente 458193 4.2
MariaEduarda 628341 7.7
MariaSophia 620940 4.5
Bruno 98196 1.7
LucasGabriel 818848 4.3
JoaoLucas 482308 4.9
Anthony 53491 6.8
Ian 389548 3.2
PedroLucas 29554 1.9
Yuri 742856 0.6
Alice 909106 2.8
Nicolas 95577 4.3
MariaLuiza 250939 5.7
MariaCecilia 216662 2.9
Otavio 530885 2.1
Bianca 727502 8.5
AnaLuiza 923304 8.0
PedroMiguel 729135 9.4
Pedro 165557 1.7
MariaValentina 699287 5.9
Giovanna 948143 6.1
Cecilia 338255 3.4
Esther 73292 4.2
Isabelly 946408 5.5
Livia 742000 2.9
Nicole 864716 2.2
AnaLiz 705538 3.5
Malu 470627 1.6
JoaoGuilherme 408922 9.8
Isaac 306136 6.9
Francisco 836236 9.7
Daniel 964576 8.9
Maitê 787156 1.7
Olivia 249372 6.4
Eduarda 744839 9.1
Victor 973209 8.9
Pietro 669928 4.0
Maya 489761 0.8
MariaFlor 227231 8.4
Antonia 347172 4.6
Melissa 463141 6.3
Mirella 657959 1.3
Henrique 143570 5.2
Rafaela 90329 9.2
AnaClara 16419 4.0
Luna 532154 4.1
Mariah 772087 5.6
Isis 975522 9.5
Yago 242101 0.9
Miguel 53276 9.9
Emanuel 678959 2.5
Lucca 915621 3.6
Emilly 800124 7.1
Theo 145318 9.7
Liz 311008 8.9
AnaSophia 906690 4.9
AnaJulia 936268 4.8
MariaVitoria 847816 2.2
Vitoria 231092 4.2
Luan 250690 6.5
Lorena 801267 3.6
Bernardo 453687 7.5
MariaHelena 992193 6.7
ArthurHenrique 559347 1.1
Marina 792298 5.7
Gael 800320 2.0
Isabella 958544 2.1
Helena 504400 2.5
Antonella 69328 9.8
Sophia 286920 2.4
DaviLucca 707482 2.3
JoaoMiguel 585604 1.2
MariaLaura 793481 5.6
Raul 671668 3.6
DaviLucas 656997 9.3
LuizMiguel 223679 3.0
Bryan 973639 6.1
Louise 263152 6.3
Carolina 82543 8.6
AnaVitoria 701396 4.8
Augusto 27542 7.4
ArthurMiguel 514691 8.6
Felipe 450433 2.9
Alicia 602891 2.8
MariaClara 952286 1.6
Elisa 79530 4.2
Kauê 854857 6.8
Arthur 94110 2.7
Thomas 131151 7.1
Eloa 927075 9.8
Thiago 919348 3.6
Agatha 795769 6.6
Ana 919744 7.7
Isadora 156378 3.4
Caio 105751 9.5
Larissa 285418 1.1
MariaHeloisa 37481 8.3
Gabriel 852884 7.5
Mariana 436544 1.9
Stella 998969 5.3
LuizFelipe 443308 2.2
Eduardo 592211 2.7
Samuel 383028 2.6
Caua 594056 0.8
Allana 235570 7.6
Danilo 158754 5.4
MariaIsis 863746 3.0
Breno 476808 2.5
Henrique 697683 8.5
Vinicius 622651 2.4
Ryan 103077 8.9
Noah 393598 5.5
LuizHenrique 480306 1.7
Isabel 703060 4.7
Beatriz 455364 4.5
Lavinia 576291 6.6
Martin 326039 6.0
AnaBeatriz 663236 7.1
Bruna 711913 8.0
Miguel 513755 5.7
Lorenzo 412028 1.8
Heitor 417735 9.5
Rodrigo 691599 8.5
Laura 672872 4.6
JoaoPedro 765088 5.2
Enrico 605105 2.1
Heloisa 624516 9.5
Julia 662454 6.0
Murilo 961270 2.9
Fernanda 361166 3.0
Valentina 232224 4.7
Gabrielly 397715 6.7
ArthurGabriel 22604 1.3
Lara 971476 2.0
Bento 172903 5.6
Joaquim 757699 1.5
Benjamin 739894 9.9
Leticia 201973 7.6
Guilherme 523227 3.6
Tomas 233989 3.1
Anthony 511246 2.5
Sarah 198220 5.7
Rafael 434924 6.4
MariaFernanda 79612 9.8
