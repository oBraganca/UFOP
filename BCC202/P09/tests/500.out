Isabel 306940 0.6
Bianca 584717 0.6
Yuri 742856 0.6
Kaique 47101 0.7
Lais 255580 0.8
Maya 489761 0.8
Caua 594056 0.8
DaviLucca 605502 0.8
Nathan 670471 0.8
Gael 156161 0.9
Yago 242101 0.9
Catarina 314370 0.9
Miguel 463864 0.9
MariaSophia 481058 0.9
Marina 783655 0.9
Caio 247076 1.0
Heloise 263375 1.0
LuizMiguel 375774 1.0
Antonio 389671 1.0
Lorenzo 508404 1.0
Heitor 540726 1.0
AnaBeatriz 571350 1.0
MariaClara 275020 1.1
Larissa 285418 1.1
ArthurHenrique 559347 1.1
Mariah 172403 1.2
JoaoLucas 217875 1.2
Arthur 365522 1.2
MariaVitoria 485389 1.2
JoaoMiguel 585604 1.2
Rodrigo 965140 1.2
ArthurHenrique 986304 1.2
Stella 825 1.3
ArthurGabriel 22604 1.3
Beatriz 105783 1.3
Enrico 299318 1.3
Luana 471718 1.3
Mirella 657959 1.3
Alicia 796322 1.3
Gabriela 182231 1.4
Gabriel 631960 1.4
Luana 926077 1.4
MariaFlor 139592 1.5
Maria 200853 1.5
Mirella 566018 1.5
Joaquim 757699 1.5
Anthony 853302 1.5
VictorHugo 960395 1.5
Anthony 22709 1.6
Yago 83271 1.6
Malu 470627 1.6
Joao 753046 1.6
Analu 867155 1.6
MariaClara 952286 1.6
Bruno 98196 1.7
Pedro 165557 1.7
AnaBeatriz 209835 1.7
LuizHenrique 480306 1.7
Anthony 569381 1.7
Maitê 787156 1.7
Beatriz 49072 1.8
Lorenzo 412028 1.8
Vitoria 468010 1.8
Isabel 623833 1.8
Enzo 978062 1.8
PedroLucas 29554 1.9
LuizOtavio 197081 1.9
Gael 252232 1.9
Mariana 436544 1.9
Sophie 57449 2.0
Leonardo 158253 2.0
Milena 260906 2.0
Heloisa 318507 2.0
AnaClara 499549 2.0
Gael 800320 2.0
Lara 971476 2.0
Kauê 85760 2.1
Emanuel 254280 2.1
MariaJulia 479915 2.1
Otavio 530885 2.1
ArthurHenrique 542641 2.1
Enrico 605105 2.1
Isabella 958544 2.1
LuizFelipe 443308 2.2
MariaVitoria 847816 2.2
Oliver 849521 2.2
Nicole 864716 2.2
Bryan 68875 2.3
Lucas 193236 2.3
Fernanda 578638 2.3
DaviLucca 707482 2.3
Otavio 800109 2.3
Matheus 840003 2.3
Clara 954235 2.3
Andre 281154 2.4
Sophia 286920 2.4
Vinicius 622651 2.4
Lucca 673398 2.4
Ayla 932012 2.4
Breno 476808 2.5
Helena 504400 2.5
Allana 509833 2.5
Anthony 511246 2.5
MariaVitoria 534268 2.5
Emanuel 678959 2.5
MariaLaura 879273 2.5
Benjamin 88894 2.6
AnaSophia 274017 2.6
AnaVitoria 315378 2.6
Samuel 383028 2.6
Pietra 532292 2.6
Arthur 94110 2.7
Gabrielly 120621 2.7
Eduardo 592211 2.7
Amanda 871304 2.7
LuizHenrique 42263 2.8
Alicia 602891 2.8
Alice 909106 2.8
Esther 916511 2.8
MariaCecilia 216662 2.9
Antonia 408719 2.9
Felipe 450433 2.9
Carolina 622297 2.9
Livia 742000 2.9
Raul 853197 2.9
Murilo 961270 2.9
Theo 215 3.0
LuizMiguel 223679 3.0
JoaoMiguel 268427 3.0
Amanda 279304 3.0
Leonardo 290864 3.0
Fernanda 361166 3.0
Sarah 620648 3.0
Analu 666012 3.0
Joaquim 760615 3.0
MariaIsis 863746 3.0
Fernando 18652 3.1
Sarah 97615 3.1
Tomas 233989 3.1
ArthurMiguel 332721 3.1
Vitoria 396525 3.1
AnaLaura 665930 3.1
Laura 767909 3.1
Murilo 12318 3.2
Ryan 182371 3.2
Ian 389548 3.2
Ayla 7034 3.3
LuizFelipe 21333 3.3
Antonella 259195 3.3
Victor 579030 3.3
EnzoMiguel 751973 3.3
Heitor 76947 3.4
Isadora 156378 3.4
MariaCecilia 215556 3.4
Ryan 296821 3.4
MariaHeloisa 327673 3.4
Cecilia 338255 3.4
Luiza 861525 3.4
Aurora 6519 3.5
AnaLiz 705538 3.5
Julia 38547 3.6
Samuel 373314 3.6
Olivia 402614 3.6
Guilherme 523227 3.6
Raul 671668 3.6
EnzoGabriel 686587 3.6
Agatha 775468 3.6
Lorena 801267 3.6
Breno 911259 3.6
Lucca 915621 3.6
Thiago 919348 3.6
Sophia 437249 3.7
Bernardo 729396 3.7
Bryan 14110 3.8
Lais 281968 3.8
LuizMiguel 407826 3.8
Rafaela 857337 3.8
Lorena 897218 3.8
Milena 818216 3.9
AnaClara 16419 4.0
Thiago 127497 4.0
Tomas 182589 4.0
Nicolas 191832 4.0
Vinicius 208375 4.0
Tomas 214777 4.0
Pietro 669928 4.0
JoaoGuilherme 684578 4.0
MariaHeloisa 868477 4.0
Eduardo 969596 4.0
ArthurMiguel 76880 4.1
PedroMiguel 133326 4.1
LucasGabriel 198874 4.1
Felipe 268577 4.1
Murilo 304314 4.1
Luna 532154 4.1
Sophie 795858 4.1
Esther 73292 4.2
Elisa 79530 4.2
LuizFelipe 85013 4.2
Vitoria 231092 4.2
Helena 350832 4.2
Vicente 458193 4.2
Joaquim 532999 4.2
DaviLuiz 617745 4.2
AnaSophia 757983 4.2
Caua 846842 4.2
Isabella 859251 4.2
Nicolas 95577 4.3
DaviLuiz 342650 4.3
JoaoMiguel 483528 4.3
LucasGabriel 818848 4.3
Pedro 971657 4.3
Francisco 263168 4.4
Caleb 337450 4.4
MariaHelena 62255 4.5
Catarina 139381 4.5
Kaique 300183 4.5
Beatriz 455364 4.5
MariaSophia 620940 4.5
Antonia 347172 4.6
Leticia 581872 4.6
MariaFernanda 582695 4.6
Laura 672872 4.6
Benjamin 966632 4.6
Valentina 232224 4.7
MariaLaura 380292 4.7
Isabel 703060 4.7
Valentina 817300 4.7
Miguel 93098 4.8
Giovanna 249151 4.8
AnaVitoria 701396 4.8
AnaJulia 936268 4.8
Yasmin 347740 4.9
Gabrielly 444280 4.9
JoaoLucas 482308 4.9
AnaVitoria 718288 4.9
LuizHenrique 904421 4.9
AnaSophia 906690 4.9
Martin 944273 4.9
Vicente 285634 5.0
DaviMiguel 380589 5.0
PedroHenrique 682653 5.0
Jose 913945 5.0
Isaac 215812 5.1
Allana 345012 5.1
Levi 749091 5.1
PedroLucas 917197 5.1
Henrique 143570 5.2
Nicole 398592 5.2
JoaoPedro 422574 5.2
Caio 507040 5.2
Lara 736579 5.2
JoaoPedro 765088 5.2
Daniel 789850 5.2
LuizGustavo 807041 5.2
JoaoVictor 810344 5.2
Thiago 953881 5.2
EnzoGabriel 419192 5.3
Thomas 763560 5.3
Clara 925424 5.3
Stella 998969 5.3
Danilo 158754 5.4
Erick 196948 5.4
Noah 393598 5.5
Sophia 557268 5.5
Enrico 576215 5.5
Kauê 723631 5.5
Benicio 822726 5.5
Emanuelly 887006 5.5
Isabelly 946408 5.5
Eloa 36760 5.6
Gabriela 52415 5.6
DaviMiguel 76260 5.6
Bento 172903 5.6
Yuri 390776 5.6
Maria 474286 5.6
Larissa 529489 5.6
DaviLucca 605485 5.6
Henrique 664983 5.6
Mariah 772087 5.6
MariaJulia 779846 5.6
MariaLaura 793481 5.6
Sarah 198220 5.7
MariaLuiza 250939 5.7
Fernanda 501011 5.7
Miguel 513755 5.7
Nathan 531058 5.7
Caua 743841 5.7
Heloisa 758760 5.7
Marina 792298 5.7
CarlosEduardo 818253 5.7
Bento 841209 5.7
Lavinia 114098 5.8
Bruna 367688 5.8
Oliver 471808 5.8
Augusto 638511 5.8
Aurora 940893 5.8
MariaValentina 699287 5.9
Martin 326039 6.0
MariaAlice 590594 6.0
Julia 662454 6.0
Rebeca 255848 6.1
Isadora 548376 6.1
MariaIsis 784649 6.1
Giovanna 948143 6.1
Bryan 973639 6.1
Liz 41739 6.2
Caleb 198040 6.2
Maitê 19930 6.3
Theo 142115 6.3
CarlosEduardo 243465 6.3
AnaJulia 258679 6.3
Louise 263152 6.3
EnzoMiguel 280455 6.3
Rafael 449601 6.3
Melissa 463141 6.3
Helena 191184 6.4
Olivia 249372 6.4
Rafael 434924 6.4
Eduarda 174322 6.5
Luan 250690 6.5
Jose 158805 6.6
Gabriel 191698 6.6
Bruna 259800 6.6
MariaAlice 430138 6.6
Lavinia 576291 6.6
AnaCecilia 744093 6.6
AnaLuiza 746474 6.6
Gabriel 767040 6.6
Agatha 795769 6.6
Luan 949278 6.6
Ana 1734 6.7
Yago 121352 6.7
Gabrielly 397715 6.7
Lorena 881917 6.7
MariaHelena 992193 6.7
Anthony 53491 6.8
Luiza 104507 6.8
Emanuelly 132580 6.8
Noah 199962 6.8
ArthurGabriel 285872 6.8
Isabella 450496 6.8
Kauê 854857 6.8
Isabelly 31069 6.9
Gabriel 292331 6.9
Isaac 306136 6.9
Emilly 466521 6.9
AnaCecilia 100446 7.0
Melissa 984326 7.0
Lara 12183 7.1
Thomas 131151 7.1
Manuela 401810 7.1
Joana 471655 7.1
Manuela 595399 7.1
Ruan 606890 7.1
AnaBeatriz 663236 7.1
Emilly 800124 7.1
Luan 887924 7.1
Larissa 671603 7.2
ArthurGabriel 250328 7.3
Antonio 510353 7.3
Augusto 27542 7.4
Clarice 231717 7.4
Henry 558152 7.4
Joao 568961 7.4
Bernardo 453687 7.5
Louise 563521 7.5
Stella 635060 7.5
Gabriel 852884 7.5
Rodrigo 960450 7.5
Cecilia 981018 7.5
Leticia 201973 7.6
Allana 235570 7.6
Emanuel 360692 7.6
Henrique 400175 7.6
Gustavo 842329 7.6
Miguel 50477 7.7
Liz 110127 7.7
Yasmin 243147 7.7
AnaLivia 410806 7.7
Breno 521330 7.7
MariaEduarda 628341 7.7
Noah 691971 7.7
Guilherme 697944 7.7
Pietro 828771 7.7
Ana 919744 7.7
Miguel 75109 7.8
MariaEduarda 97285 7.8
Livia 656287 7.8
Guilherme 78275 7.9
Carolina 320126 7.9
Mariana 344745 7.9
Martin 306197 8.0
AnaLaura 362626 8.0
Gustavo 595974 8.0
Julia 703209 8.0
Bruna 711913 8.0
Rebeca 836059 8.0
JoaoGabriel 855921 8.0
AnaLuiza 923304 8.0
MariaClara 950345 8.0
Davi 143083 8.1
Isadora 702044 8.1
Luna 718735 8.1
Alicia 762996 8.1
Heloise 141848 8.2
VictorHugo 377592 8.2
Eloa 412970 8.2
AnaJulia 529241 8.2
MariaHeloisa 37481 8.3
Lucas 320485 8.3
Leticia 330354 8.3
Marina 447724 8.3
Davi 760183 8.3
Lavinia 838900 8.3
Maya 985540 8.3
MariaFlor 227231 8.4
Lorenzo 366023 8.5
Felipe 516131 8.5
Rodrigo 691599 8.5
Henrique 697683 8.5
Bianca 727502 8.5
Carolina 82543 8.6
DaviLucas 457249 8.6
ArthurMiguel 514691 8.6
Rafael 535828 8.6
Fernando 622886 8.6
MariaLuiza 892017 8.6
Erick 949111 8.6
Ruan 295172 8.7
Thomas 541886 8.7
MariaIsis 822458 8.7
JoaoVictor 88463 8.8
Matheus 318725 8.8
Malu 641153 8.8
Ana 765971 8.8
Joana 947751 8.8
JoaoPedro 27084 8.9
Ryan 103077 8.9
Liz 311008 8.9
LuizOtavio 411197 8.9
JoaoGabriel 601303 8.9
Clarice 641685 8.9
MariaHelena 809404 8.9
Levi 926862 8.9
Benicio 929061 8.9
Daniel 964576 8.9
Victor 973209 8.9
AnaLivia 36809 9.0
Raul 192002 9.0
Bruno 416267 9.0
Laura 564090 9.0
Henry 570628 9.0
Elisa 729894 9.0
Valentina 773133 9.0
Arthur 55242 9.1
Isis 506111 9.1
MariaFernanda 688453 9.1
Eduarda 744839 9.1
Samuel 881887 9.1
Mariana 928332 9.1
Rafaela 90329 9.2
DaviLucas 201464 9.2
Lucca 366467 9.2
Andre 737240 9.2
Augusto 396061 9.3
Alice 631097 9.3
DaviLucas 656997 9.3
Henrique 199728 9.4
PedroMiguel 729135 9.4
Caio 105751 9.5
LuizGustavo 366151 9.5
Heitor 417735 9.5
Danilo 504032 9.5
Pietra 591628 9.5
Heloisa 624516 9.5
AnaLiz 765124 9.5
Eduardo 789299 9.5
Enzo 848100 9.5
Danilo 894641 9.5
Isis 975522 9.5
Louise 870632 9.6
Theo 145318 9.7
Antonella 270313 9.7
Ian 474641 9.7
Vinicius 686440 9.7
Francisco 836236 9.7
Antonella 69328 9.8
MariaFernanda 79612 9.8
MariaValentina 229209 9.8
JoaoGuilherme 408922 9.8
Emilly 575184 9.8
Eloa 927075 9.8
Elisa 11187 9.9
Miguel 53276 9.9
PedroHenrique 323568 9.9
Bento 683393 9.9
Benjamin 739894 9.9
Agatha 609677 10.0
Bernardo 974729 10.0
